frontend/src/component/common/Search/SearchSuggestions/SearchDescription/SearchDescription.tsx

```diff
@@ -61,7 +61,7 @@ export const SearchDescription: VFC<ISearchDescriptionProps> = ({
                                     {filter.values.join(',')}
                                 </StyledCode>{' '}
                                 in {filter.header}. Options:{' '}
-                                {filter.options.join(', ')}
+                                {filter.options.slice(0, 10).join(', ')}
```

Comment:
we don't want to present more than 10 options. E.g. for feature names in change request list